docs/gateway-api/gateway-api-v1.md

```diff
@@ -228,22 +250,31 @@ AKO accepts the following Gateway configuration for this release:
   2. Gateway MUST NOT contain protocols other than HTTP or HTTPS.
   3. Gateway MUST contain a hostname. Hostname as `*` is not supported and `*.domain` is supported.
   4. Gateway MUST NOT contain TLS modes other than `Terminate`.
+  5. If the Secret specified in Gateway-> listeners-> TLSConfig does not exist or is in a namespace other than the Gateway namespace, AKO Gateway implementation will attach `SystemDefaultCert` to the Gateway. Once Secret is created in the same namespace as that of Gateway, the `certRef` will be updated accordingly.
+  6. Gateway CAN have multiple listeners with same/overlapping hostname. 
+  7. Two Gateways MUST NOT have listeners with same/overlapping hostname.
+  8. `Gateway-> listeners-> allowedRoutes-> namespaces-> from` with value `selector` is not supported.
```

Comment:
Can we re-phrase 7 and 8, combine it together. Reason: if `FromNamespaces` field has value `selector` only then `Routenamespaces.selector`  will be there.

Reply:
We must not combine 7 and 8 since they are not related.

Reply:
@pkoshtavmware my bad. I wanted to say 8 and 9.